gui/create-item.lua

```diff
@@ -36,10 +36,10 @@ function getCreaturePartList(creatureID, casteID)
 end
 
 function getCreaturePartLayerList(creatureID, casteID, partID)
- local crplList={}
+ local crplList={{"whole"}}
  for k,crpl in ipairs(df.global.world.raws.creatures.all[creatureID].caste[casteID].body_info.body_parts[partID].layers) do
   local str = crpl.layer_name
-  table.insert(crplList,{str})
+  table.insert(crplList,{string.lower(str)}) -- lowercase string for easier searching
```

Comment:
let's actually not do this here. we have a pending action to make FilteredList widgets search case insensitive by default. Let's not lose the displayed capitalization in these prompts.

Reply:
alright